### DISCUSSION TITLE
so-elastalert - Missing state

### DISCUSSION THREAD

**Discussion by a3hiennt** [Security-Onion-Solutions/securityonion#5240] [General]
Hi, I'm facing issue related to so-elastalert.
It showing MISSING state
![image](https://user-images.githubusercontent.com/80515844/130315253-02085606-bf43-4e7c-be93-d5c61cdcbe3d.png)
I tried to restart this by so-elastalert-restart but it still not working.
Please help me check.
Thanks

**Comment by a3hiennt** [comment]
This is docker logs of command "docker logs so-elastalert"
![image](https://user-images.githubusercontent.com/80515844/130315511-240e8b9e-e249-4e6d-824f-0c4b0f073086.png)

**Reply by wally-kroeker** [reply]
did you ever resolve this problem.  I am having the same issue it was there on first install but now it is not.  The docker container is completely missing

**Reply by wally-kroeker** [reply]
I was able to figure out what happened.  I had created a new test rule in the /opt/so/rules/elastalert/playbook directory.  I delete the yaml file and ran so-elastalert-start and it came up

**Reply by dougburks** [reply]
@wally-kroeker In the future, please start a new discussion rather than replying to somebody else's discussion with your problem. From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and in that new discussion you can provide a hyperlink to the related discussion.
